plone/namedfile/scaling.py

```diff
@@ -192,18 +193,18 @@ def create_scale(self, data, direction, height, width, **parameters):
         )
 
     def __call__(
-        self,
-        fieldname=None,
-        direction='thumbnail',
-        height=None,
-        width=None,
-        scale=None,
-        **parameters
+            self,
```

Comment:
Actually, just because my linter said so.
The reason for this formatting is, that this way it isn't on the same level like the inner function block code.
Although that's also just a question of taste.